content/app/development/ux/texts/_index.en.md

```diff
@@ -113,6 +113,40 @@ The texts are stored in `App/config/texts`
 
 ![Altinn Studio Repos](edit-texts-in-repos.png "Change texts in Altinn Studio Repos")
 
+## Change dafault texts and error messages in an application
+
+It is possible to change default texts and error messages that is displayed in the app. 
+Here are the id:s with its default value in [English](https://github.com/Altinn/app-frontend-react/blob/v3/src/language/texts/en.ts), 
+[Norwegian Bokmål](https://github.com/Altinn/app-frontend-react/blob/v3/src/language/texts/nb.ts) and [Nynorsk](https://github.com/Altinn/app-frontend-react/blob/v3/src/language/texts/nn.ts) 
```

Comment:
I think we could safely link to the main branch here, as v3 is on its way out.

Also, not sure which notation `id:s` is, but they could probably be called `keys` or just `IDs`. Look here for more: https://english.stackexchange.com/questions/212040/is-the-plural-form-of-id-spelled-ids-or-id